sdl2promela/stop_condition/translator.py

```diff
@@ -1018,6 +1022,40 @@ def _generate_entry_loop() -> promela.Do:
     )
 
 
+def _generate_else_alternative(
+    input_model: model.StopConditionModel, context: GenerateContext
+) -> promela.Alternative:
+    expressions = []
+
+    # Collect all expressions
```

Comment:
all invariant? There is no filter out and eventualy...

Reply:
This should be removed